### ISSUE TITLE
nan values

### ISSUE THREAD

**Issue by avipersin** [jakevdp/mpld3#317]
I am using the master branch with Python 3.4 and am having a problem with NaN values. The problem only exists when trying to display the graph from fig_to_html() but when calling show() it works fine.

I saw there was a previous thread and test addressing this issue but I'm not sure where that went. #16  

when I call mpld3.fig_to_html(plt.gcf()) and try to render it on a web page I am getting this: Error: Invalid value for <path> attribute transform="translate(478.2857142857143,NaN)".  Every time I drag the plot I get an additional error of the same sort printed to console.  

When I remove the NaN value though, everything works fine.

**Comment by jakevdp** [comment]
Yes, this is a bug. We wrote special cases to address NaN values in data, but apparently we also need a special case to address NaN values in transforms.

If you feel like tackling this, contributions are welcome – thanks!

**Comment by avipersin** [comment]
I would, but I just started using this library and don't yet feel confident enough with the underpinnings to fix bugs.

As I become more comfortable, I would love to help with these kinds of things. I think this library has huge potential and is very useful especially for the scientific community.
